api/tableau_api/wdc.py

```diff
@@ -0,0 +1,43 @@
+from flask import render_template
+from flask.views import MethodView
+from flask_cors import cross_origin
+
+from api.tableau_api.constants import SERIALIZABLE_FIELD_NAMES, FIELD_TYPE_MAP
+from database.tableau_api_models import SummaryStatisticDaily
+
+class WebDataConnector(MethodView):
+
+    path = '/api/v0/studies/<string:study_id>/summary-statistics/daily/wdc'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # build the columns datastructure for tableau to enumerate the format of the API data
+        fields = SummaryStatisticDaily._meta.get_fields()
+        self.cols = '[\n'
+        for field in fields:
+            if field.name in SERIALIZABLE_FIELD_NAMES:
+                for (py_type, tableau_type) in FIELD_TYPE_MAP:
+                    if isinstance(field, py_type):
+                        self.cols += f"{{id: '{field.name}', dataType: {tableau_type},}},\n"
+                        # ex line: {id: 'participant_id', dataType: tableau.dataTypeEnum.int,},
+                        break
+                else:
+                    # if the field is not recognized, supply it to tableau as a string type
+                    self.cols += f"{{id: '{field.name}', dataType: tableau.dataTypeEnum.string,}},\n"
```

Comment:
If we change `FIELD_TYPE_MAP` to be a dict instead, this can be simplified a bunch.